package-lock.json

```diff
@@ -33962,7 +33938,7 @@
       "version": "2.1.2",
       "resolved": "https://registry.npmjs.org/safer-buffer/-/safer-buffer-2.1.2.tgz",
       "integrity": "sha512-YZo3K82SD7Riyi0E1EQPojLz7kpepnSQI9IyPbHHg1XXXevb5dJI7tpyN2ADxGcQbHG7vcyRHk0cbwqcQriUtg==",
-      "devOptional": true
+      "dev": true
     },
     "sane": {
       "version": "4.1.0",
```

Comment:
Were any of these package-lock.json changes in respect to the removing of the error message? Or are these separate with the changing of your node version specifically? If the later, this probably shouldn't be committed

Reply:
This is in regard to the entire file in general. Not just this one line

Reply:
These changes were made after the upgrade to node, I will go back and  make sure this is not in the commits.